jimmy/modules/node_envvars/resources/schema.yaml

```diff
@@ -0,0 +1,8 @@
+title: Schema for Jenkins Global EnvVars setter
+type: object
+minProperties: 1
+additionalProperties:
```

Comment:
Could you please look here https://spacetelescope.github.io/understanding-json-schema/reference/object.html#pattern-properties ?
AFAIU, hee should be KV with UpperString to string mapping. But now schema allows to pass something like
```
1: {"foo": {"bar": ["baz" ,2]}}
```

Reply:
I believe that using

      additionalProperties:
        type: string

is actually enough to ensure that the value must be a string.
This is shown by the last example on the doc page you've linked.

I've also made [a gist](https://jsonschemalint.com/#/version/draft-06/markup/json?gist=135b6f50848199d36a3eeedcafbca1ca) to illustrate this.

Actually, while I've reviewed the schema I've suddenly realized that setting the minimum of one child element for a config to be considered valid will lead to an inability to disable the environment variables at all. 
This makes `{node-envvars: {"node1": []}}` invalid, while it represents a valid state of environment variables. I'm going to fix that on both PRs.